docs/tutorial.rst

```diff
@@ -248,6 +248,20 @@ response, and for each one, the response contains the following info:
   access_token when it expires. ``None`` unless explicitly requested.
   Details on refresh_token are in the next section
 
+.. note::
+
+    The keys into ``by_resource_server`` are the registered ``resource_server``
+    value for the service.
+    In general, the ``resource_server`` value will by the ID of the client used
+    in a service.
+
+    However, for Globus services like Globus Auth and Globus Transfer, the
+    ``resource_server`` has been customized to be the hostname for the service.
+
+    You can lookup the ``resource_server`` for most globus-sdk supported
+    clients by looking at the ``resource_server`` class attribute, e.g.
```

Comment:
```suggestion
    clients by looking at the ``resource_server`` class attribute, e.g.,
```

Reply:
Oooh, I am definitely not consistent about that! Fixed.